src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -34,18 +35,19 @@ public String test() {
     public String validateUser(@RequestParam String userId, @RequestParam String password, HttpSession session) {
 
         log.info("userId, password [{}][{}]", userId, password);
-        User dbUser = userRepository.findUserWithMatchedPassword(userId, password);
+        Optional<User> dbUser = userRepository.findUserWithMatchedPassword(userId, password);
 
-        if (!dbUser.getUserId().equals(userId) || !dbUser.getPassword().equals(password)) {
+        if (dbUser.isEmpty()) {
```

Comment:
```suggestion
        if (!dbUSer.isPresent()) {
```
옵셔널을 잘 모르면 중요한 내용이니 이번 기회에 공부를 해 보는 것도 좋을 것 같아요.